egapro/views.py

```diff
@@ -189,6 +189,15 @@ def is_declaration(self, response, data):
         return True
 
 
+@app.route("/index/{siren}")
+async def get_by_siren(request, response, siren):
+    declarations = await db.declaration.all_by_siren("12345678")
+    response.json = {
+        d["year"]: models.Data(d)["data"]["declaration"]["noteIndex"]
```

Comment:
there is a shortcut `.grade`, see models.py

Reply:
implemented :+1: